README.md

```diff
@@ -58,8 +73,11 @@ Load the Recurly library files and set your subdomain and API Key globally:
 <?php
 require_once('./lib/recurly.php');
 
+/* https://<your-subdomain>.recurly.com */
 Recurly_Client::$subdomain = 'your-subdomain';
-Recurly_Client::$apiKey = '012345678901234567890123456789ab';
+/* your private API key */
+Recurly_Client::$apiKey = '012345678901234567890123456789ab'; 
+>
```

Comment:
so the `>` seems like part of a `?>` which is now [typically omitted from PHP files](http://stackoverflow.com/questions/4410704/why-would-one-omit-the-close-tag). so let's drop the last line.